### ISSUE TITLE
Stylus seen as a mouse, so no pressure

### ISSUE THREAD

**Issue by tubbadu** [H-M-H/Weylus#239]
I searched across all open issues and did not find anything related, sorry if I missed some.
OS: EndeavourOS running Weylus latest release from github, Fedora 37 on a reversible 2in1 laptop pc running both firefox and chromium-freeworld. Both with KDE Wayland session.

The stylus is seen as a simple mouse: if I disable the mouse input checkbox (with all other checked), it stops working at all. If kept enabled it works, but there is no pressure recognition, because it is seen as a simple mouse click and drag. I think it's a problem of the browser, but I'm using the latest version available on my distribution of both firefox and chromium-freeworld (I also tried with Falkon and Angelfish, with the same exact result). 

Is it a problem of these browsers, of my setup, or a weylus problem?

**Comment by Diordany** [comment]
Did you set up the `uinput` interface? If not, see [this](https://github.com/H-M-H/Weylus#linux) section (also tick the `Enable uinput` box on your tablet).

**Comment by Diordany** [comment]
> @Diordany I found that step of the setup instructions oddly placed, but I understood that it was only meant to be used if reverting/uninstalling.

Yeah, so we can rule that out in your case. I agree with your point, someone can easily mistake that command for being part of the setup process.

> Another thing I found weird about the instructions was
> 
> > Then, either reboot, or run
> 
> and the first time I tried setting this up I may have both run those commands _and_ rebooted. But just in case that was the cause of my issue I ended up uninstalling (i.e. removing the uinput group, deleting the .rules file), rebooting and then going through the setup again, taking care to only do one.

Yes, I also agree with this.

**Comment by Diordany** [comment]
I read earlier today that this project isn't being maintained right now (#255). I'm also not a contributor myself, but if I find anything that might help solve your problem, I'll let you know @everdred .
